eagle-core/eagle-alert-parent/eagle-alert/alert-common/src/main/java/org/apache/eagle/alert/config/ConfigBusConsumer.java

```
@@ -54,7 +54,12 @@ public ConfigBusConsumer(ZKConfig config, String topic, ConfigChangeCallback cal
 
     public ConfigValue getConfigValue() throws Exception {
         byte[] value = curator.getData().forPath(zkPath);
-        ConfigValue v = mapper.readValue(value, ConfigValue.class);
+        ConfigValue v;
+        try {
+            v = mapper.readValue(value, ConfigValue.class);
+        } catch (JsonParseException e) {
+            v = new ConfigValue();
```

Pls. add `LOG.warn(e.getMessage())` for the ignored exception.